Assets/MixedRealityToolkit.Tests/PlayModeTests/StandardShaderTests.cs

```diff
@@ -0,0 +1,129 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+#if !WINDOWS_UWP
+// When the .NET scripting backend is enabled and C# projects are built
+// The assembly that this file is part of is still built for the player,
+// even though the assembly itself is marked as a test assembly (this is not
+// expected because test assemblies should not be included in player builds).
+// Because the .NET backend is deprecated in 2018 and removed in 2019 and this
+// issue will likely persist for 2018, this issue is worked around by wrapping all
+// play mode tests in this check.
+
+using Microsoft.MixedReality.Toolkit.Utilities;
+using System.Collections;
+using UnityEngine;
+using UnityEngine.TestTools;
+
+namespace Microsoft.MixedReality.Toolkit.Tests
+{
+    public class StandardShaderTests : BasePlayModeTests
+    {
+        private const float VisualizatonWaitTime = 1.0f;
+        private readonly Vector3 LightDirection = new Vector3(-0.5f, -0.5f, 0.0f);
+
+        /// <summary>
+        /// Tests if the MeshOutline component can be added an manipulated at runtime.
+        /// </summary>
+        /// <returns></returns>
+        [UnityTest]
+        public IEnumerator TestMeshOutline()
+        {
+            var light = InstantiateDirectionalLight(LightDirection);
+            var gameObject = GameObject.CreatePrimitive(PrimitiveType.Cube);
+
+            var meshOutline = gameObject.AddComponent<MeshOutline>();
+            meshOutline.OutlineMaterial = InstantiateStandardShaderOutlineMaterial(Color.red);
+            meshOutline.OutlineWidth = 0.02f;
```

Comment:
idea: could you:

1. make the outline some color like bright green (0, 255, 0)
2. Capture camera texture
3. ensure that at least 100 pixels are bright green.

screen shot / capture camera texture and at least ensure that some

Reply:
I like that idea! Last time I tried to take screenshots on the build machine it kept failing with "missing device context" issues. But, maybe we could think of another way to accomplish this?